packages/xstate-test/src/TestModel.ts

```diff
@@ -398,19 +389,6 @@ export class TestModel<TState, TEvent extends EventObject> {
   }
 }
 
-/**
- * Specifies default configuration for `TestModel` instances for path generation options
- *
- * @param testModelConfiguration The partial configuration for all subsequent `TestModel` instances
```

Comment:
IMHO this breaks the encapsulation - it also wasn't actually utilized as per: https://github.com/statelyai/xstate/pull/3598/files#r969327028